src/Notifier.js

```diff
@@ -16,8 +16,10 @@ limitations under the License.
 
```

Comment:
I agree other apps will want to present their notifications differently. Exposing this as a React-style component though is incredibly misleading. I'm completely happy to expose an API on the `Notifier` to allow other apps to configure the look and feel as they wish.